### PR TITLE
Spawn player and update time

### PR DESCRIPTION
**Pull request by Thomas-Harit**

### CODE REVIEW

File: cubic-server/Default/DefaultWorld.cpp
Comment (huntears) [inline cubic-server/Default/DefaultWorld.cpp:39]: The clients will be added by #48 , so maybe we can wait for that to be merged before merging this

File: cubic-server/Dimension.cpp
Comment (huntears) [inline cubic-server/Dimension.cpp:24]: Same here, maybe waiting for #48 might be a good idea

File: protocol/typeSerialization.hpp
Comment (huntears) [inline protocol/typeSerialization.hpp:289]: Why not just use addLong two times ?
#48 also adds this function so once again, I think waiting for merge is the good move.

File: cubic-server/Default/DefaultWorld.hpp
Comment (huntears) [reply cubic-server/Default/DefaultWorld.hpp:15]: Looking at wiki.vg it looks like I was wrong and the time is indeed per world, not per dimensions

**Review by huntears — DISMISSED** [review]

**Review by huntears — DISMISSED** [review]

**Review by STMiki — DISMISSED** [review]

**Review by huntears — DISMISSED** [review]

**Review by huntears — APPROVED** [review]

**Review by STMiki — APPROVED** [review]

**Pull request merged**
